ga4gh/cli.py

```diff
@@ -76,13 +76,16 @@ def server_main(parser=None):
 ##############################################################################
 
 
-class AbstractSearchRunner(object):
+class AbstractQueryRunner(object):
     """
-    Abstract base class for search runner classes
+    Abstract base class for runner classes
```

Comment:
I'm with @dcolligan here; this is Python, we don't need a compiler to tell us what's good and bad!